runners/google-cloud-dataflow-java/pom.xml

```diff
@@ -122,6 +122,50 @@
         </plugins>
       </build>
     </profile>
+
+    <!-- A profile that adds an integration test phase for streaming if and only if
+         the streamingValidatesRunnerPipelineOptions maven property has been set.
+         It should be set to a valid PipelineOptions JSON string. -->
+    <profile>
+      <id>streaming-validates-runner-tests</id>
+      <activation>
+        <property><name>streamingValidatesRunnerPipelineOptions</name></property>
+      </activation>
+      <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-surefire-plugin</artifactId>
+              <executions>
+                <!-- Execution in addition to the default streaming-validates-runner-tests execution -->
+                <execution>
+                  <id>streaming-validates-runner-tests</id>
+                  <phase>integration-test</phase>
+                  <goals>
+                    <goal>test</goal>
+                  </goals>
+                  <configuration>
```

Comment:
I've merged the two executions into a single profile, but actually I'm not sure how to share the other bits of surefire config between just these two executions without affecting the others.